layouts/basic/modules/Vtiger/Edit/Field/Phone.tpl

```diff
@@ -20,7 +20,7 @@
 			{if $NUMBER}
 				{assign var="PHONE_DETAIL" value=App\Fields\Phone::getDetails($NUMBER)}
 			{else}
-				{assign var="PHONE_DETAIL" value=false}
+				{assign var="PHONE_DETAIL" value=['country' => substr(\App\Language::getLanguage(),-2)]}
```

Comment:
```suggestion
				{assign var="PHONE_DETAIL" value=['country' => \App\Language::getShortLanguageName()]}
```

Reply:
short name doesn't work, 
\App\Language::getShortLanguageName() = en
substr(\App\Language::getLanguage(),-2) = US
u_yf_countries.code =US

Reply:
thanks for the info we will check it,
if you have any examples for which it works incorrectly send us